src/gnmi_clients/src/github.com/openconfig/gnmi/CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# How to Contribute
```

Comment:
Don't we fix the commit-id for the pull?

Reply:
I have removed the entire gnmi_client directory now, and instead "go get" them and then patch as needed. The commit id is fixed.